README.md

```diff
@@ -37,21 +37,21 @@ Like any UI Library / Design System the main goal of Storefront UI is to **accel
 
 There is a lot of the things you have to take care of to build good user interfaces:
 
-🔹 **Developer/Designer workflow**: Storefront UI comes out of the box with Figma files - a pixel-perfect representation of your code that your design and engineering teams needs to stay aligned. 
+🔹 **Developer/Designer workflow.** Storefront UI comes out of the box with Figma files - a pixel-perfect representation of your code that your design and engineering teams needs to stay aligned. 
```

Comment:
```suggestion
🔹 **Developer/Designer workflow** - Storefront UI comes out of the box with Figma files - a pixel-perfect representation of your code that your design and engineering teams needs to stay aligned. 
```

I think the dashes here would add more consistency for the lists on this README. I'll just leave one comment, but it would apply to all of the list items.